vendors/st/boards/stm32l475_discovery/aws_tests/application_code/cellular_setup.c

```diff
@@ -179,6 +180,13 @@
         if( cellularStatus == CELLULAR_SUCCESS )
         {
             cellularStatus = Cellular_SetDns( CellularHandle, CellularSocketPdnContextId, testCELLULAR_DNS_SERVER_ADDRESS );
+
+            /* Adding CELLULAR_UNSUPPORTED check due to init fail. */
+            /* Modem use dynamic DNS. */
+            if( cellularStatus == CELLULAR_UNSUPPORTED )
```

Comment:
nit:
cellularStatus  will change to Success here, suggest to add some log here.
"Set DNS not support"

Reply:
This will be updated in next version.